jest.config.js

```diff
@@ -4,10 +4,14 @@
  */
 
 module.exports = {
+  testEnvironment: 'jsdom',
   clearMocks: true,
 
   collectCoverage: true,
-  collectCoverageFrom: ["<rootDir>/src/features/**/use*.{js,jsx,ts,tsx}"],
+  collectCoverageFrom: [
+    "<rootDir>/src/features/**/use*.{js,jsx,ts,tsx}",
```

Comment:
We can remove this line for now, this was taken from another project with a different set up